### DISCUSSION TITLE
Url is too long to display.

### DISCUSSION THREAD

**Discussion by dydhyhwu** [Milkdown/milkdown#452] [Q&A]
**Describe the bug**
图片上传后回显的路径， 提示`Url is too long to display.`，有配置可以调整吗？这样就没办法去改或者复制这个路径了

**To Reproduce**
url: https://image.demo.cn/p/doc/2022-04-02-16-27-27-QQ%E5%9B%BE%E7%89%8720211216151612.gif

**Expected behavior**
期望应该要能展示出来Url

**Desktop (please complete the following information):**

-   OS: windows 11
-   Browser chrome
-   Version  99.0.4844.51

**Comment by Saul-Mirone** [comment]
![image](https://user-images.githubusercontent.com/10047788/161375149-6f829867-7c0c-4452-80af-107c630507f1.png)
怎么复现

**Reply by dydhyhwu** [reply]
你 APPLY 之后再点这个图片，就显示了。

**Comment by Saul-Mirone** [comment]
如果你是拖拽上传的，默认是把图片转成base64，那肯定展示不了，太长了，你要配置下plugin-upload
https://github.com/Saul-Mirone/milkdown/blob/main/packages/plugin-upload/README.md#setup-uploader

**Reply by dydhyhwu** [reply]
拖拽上传， 然后我配置了Upload。 刚刚我又测了一下直接加链接，APPLY之后也这样了

**Comment by dydhyhwu** [comment]
![image](https://user-images.githubusercontent.com/7239380/161375985-f082abaa-a66d-4360-8da1-2b33ea3d403e.png)
![image](https://user-images.githubusercontent.com/7239380/161375994-d5b4b694-a1b1-45c7-800f-570ee9be5005.png)

**Reply by Saul-Mirone** [reply]
好像是以前的bug，现在应该修复了，可以等等下个版本

**Reply by dydhyhwu** [reply]
嗯嗯， 哪个版本号会修复这个？

**Reply by Saul-Mirone** [reply]
> 嗯嗯， 哪个版本号会修复这个？

6.0，正在beta
